apps/eas-expo-go/eas.json

```diff
@@ -5,21 +5,30 @@
   },
   "build": {
     "base": {
+      "credentialsSource": "local",
       "android": {
         "image": "ubuntu-20.04-jdk-11-ndk-r21e",
         "cache": {
           "customPaths": [
-            "../../android/ReactAndroid/prebuiltHermes"
+            "../../android/ReactAndroid/prebuiltHermes",
+            "/home/expo/.gradle/caches/7.3.3"
```

Comment:
the gradle version changes from time to time after react-native upgrade. do you have any better solution than the fixed version here?

Reply:
I forgot to remove it, it does not help much and the entire caches directory is a bit too large with our current restrictions.